packages/fgtclb/academic-persons/Classes/Command/CreateProfilesCommand.php

```diff
@@ -31,12 +32,32 @@ public function __construct(private readonly FrontendUserProvider $frontendUserP
 
     protected function configure(): void
     {
-        $this->setHelp('This command create profiles for all frontend users that do not have a profile yet but should have one.');
+        $this
+            ->setHelp('This command create profiles for all frontend users that do not have a profile yet but should have one.')
+            ->addOption(
+                'exclude-pids',
+                'e',
+                InputArgument::OPTIONAL,
+                'Comma-separated list of PIDs to exclude',
+                null
+            )
+            ->addOption(
+                'include-pids',
+                'i',
+                InputArgument::OPTIONAL,
+                'Comma-separated list of PIDs to include',
+                null
+            );
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $frontendUsers = $this->frontendUserProvider->getUsersWithoutProfile();
+        $options = $input->getOptions();
+
+        $frontendUsers = $this->frontendUserProvider->getUsersWithoutProfile(
+            array_key_exists('include-pids', $options) && is_string($options['include-pids']) ? $options['include-pids'] : null,
```

Comment:
We should unpack the comma-list of pids (include & exclude) here directly and pass them down as arrays and not as strings.